typescript/infra/src/config/agent.ts

```diff
@@ -120,6 +120,8 @@ interface BaseRelayerConfig {
   gasPaymentEnforcementPolicy: GasPaymentEnforcementPolicy;
   whitelist?: MatchingList;
   blacklist?: MatchingList;
+  transactionGasLimit?: string;
```

Comment:
any reason to not make this a number?

Reply:
A U256 does not fit in 2^53. I guess I can use bigint or whatever the official JS one was.

Reply:
The reason I was asking is bc the gas limit in practice is usually at most like tens of millions, but fwiw maybe this can be really high in weird execution environments, so I'm happy to just keep as a string